docs/source/developer/node-with-devstack.rst

```diff
@@ -0,0 +1,274 @@
+Deploying Node with DevStack and Ironic
+=======================================
+
+This covers the steps to deploy a node using Ironic and set up DevStack with Ironic integration, including creating fake nodes in Ironic.
+
+.. contents:: Table of Contents
+   :depth: 2
+   :local:
+
+Image That We Used
+------------------
+
+For this, we used the Ubuntu 22.04 image to provision a node. You can use any image that is compatible with DevStack. You can download the image from the official Ubuntu website:
+
+[Download Ubuntu 22.04](https://releases.ubuntu.com/22.04/ubuntu-22.04-live-server-amd64.iso)
+
+Deploy the Node with an Image and Attach an External Floating IP
```

Comment:
This is my fault, but we probably don't need this section; most people testing ESI won't have an ESI node available; instead they really just need an Ubuntu bare metal node or VM. I'd change the prerequisites to simply say "VM or bare metal node provisioned with..."